mysql-ballerina/build.gradle

```diff
@@ -433,28 +477,16 @@ updateTomlFile.dependsOn copyStdlibs
 startTestDockerContainer.dependsOn createTestDockerImage
 startTestSslDockerContainer.dependsOn createTestDockerImage
 
-ballerinaTest.dependsOn updateTomlFile
-ballerinaTest.dependsOn initializeVariables
-ballerinaTest.dependsOn ":${packageName}-native:build"
-ballerinaTest.dependsOn startTestDockerContainer
-ballerinaTest.dependsOn startTestSslDockerContainer
-ballerinaTest.finalizedBy revertTomlFile
-ballerinaTest.finalizedBy stopTestDockerContainer
-ballerinaTest.finalizedBy stopTestSslDockerContainer
-test.dependsOn ballerinaTest
-
-ballerinaBuild.dependsOn test
 ballerinaBuild.dependsOn updateTomlFile
 ballerinaBuild.dependsOn initializeVariables
 ballerinaBuild.dependsOn ":${packageName}-native:build"
+ballerinaTest.dependsOn startTestDockerContainer
+ballerinaTest.dependsOn startTestSslDockerContainer
 ballerinaBuild.finalizedBy revertTomlFile
 ballerinaBuild.finalizedBy stopTestDockerContainer
 ballerinaBuild.finalizedBy stopTestSslDockerContainer
+test.dependsOn ballerinaBuild
 build.dependsOn ballerinaBuild
 
-ballerinaPublish.finalizedBy revertTomlFile
-ballerinaPublish.dependsOn initializeVariables
-ballerinaPublish.dependsOn updateTomlFile
-ballerinaPublish.dependsOn ":${packageName}-native:build"
-ballerinaPublish.dependsOn ballerinaBuild
-publish.dependsOn ballerinaPublish
+publishToMavenLocal.dependsOn build
+publish.dependsOn build
```

Comment:
```suggestion
publish.dependsOn build

```

Reply:
fixed in https://github.com/ballerina-platform/module-ballerinax-mysql/pull/132/commits/ea0e93daab7f4da8146195ed056ccaf6106ba229